gcp-out-of-cluster.md

```diff
@@ -37,3 +37,5 @@ Container:  "kubernetes_container":  &lt;container>
 </pre>
 
 To use an alternative or existing label schema for GCP cloud assets, you may supply these in your [values.yaml](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml) under the "kubecostProductConfigs.labelMappingConfigs.\<aggregation\>\_external_label" 
+
+> note: Google generates its own labels for GKE resources "goog-gke-node" and "goog-gke-volume". Values with these labels are excluded from OOC costs because they are already included in in-cluster costs under the kubernetes service. This will exclude GKE resources from other clusters however, to address this it is recommended to have kubecost installed on each cluster where insights into costs are required.
```

Comment:
Some of this verbiage was a little confusing to me. How do you feel about this suggestion?

> Note: Google generates special labels for GKE resources (e.g. "goog-gke-node", "goog-gke-volume"). Values with these labels are excluded from out-of-cluster costs because Kubecost already includes them as in-cluster assets. Thus, to make sure all cloud assets are included, we recommend installing Kubecost on each cluster where insights into costs are required.

Reply:
Much more concise, I like it